lib/gui/pages/settings/templates/settings.tpl.html

```diff
@@ -7,7 +7,7 @@ <h1 class="title space-bottom-large">Settings</h1>
         ng-model="settings.currentData.errorReporting"
         ng-change="settings.toggle('errorReporting')">
 
-      <span>Report errors</span>
+      <span>Report errors back to resin.io</span>
```

Comment:
> Maybe we could also split it up into error reports & telemetry (just a thought, but that's out of the scope of this PR)?

We _could_, but I find it hard to imagine situations in which a user would enable one but disable the other?